File: pl-domain/patient-service/src/main/java/com/polylab/patients/domain/service/PatientServiceImpl.java
package com.polylab.patients.domain.service;

import com.jxareas.jxcore.domain.service.PersistentService;
import com.polylab.patients.persistence.model.Patient;
import com.polylab.patients.persistence.repository.PatientRepository;
import org.springframework.stereotype.Service;

@Service
public class PatientServiceImpl extends PersistentService<Patient, Integer> implements PatientService {
    public PatientServiceImpl(PatientRepository repository) {
        super(repository);
    }
}

File: pl-domain/patient-service/src/main/java/com/polylab/patients/domain/service/PatientServiceImpl.kt
package com.polylab.patients.domain.service

import com.jxareas.efficax.domain.service.PersistentService
import com.polylab.patients.persistence.model.Patient
import com.polylab.patients.persistence.repository.PatientRepository
import org.springframework.stereotype.Service

@Service
class PatientServiceImpl(repository: PatientRepository) :
    PersistentService<Patient, Int>(repository), PatientService
